homeassistant/components/mqtt/alarm_control_panel.py

```diff
@@ -150,8 +150,9 @@ def _setup_from_config(self, config):
         value_template = self._config.get(CONF_VALUE_TEMPLATE)
         if value_template is not None:
             value_template.hass = self.hass
-        command_template = self._config[CONF_COMMAND_TEMPLATE]
-        command_template.hass = self.hass
+        self.command_template = MqttCommandTemplate(
+            self._config[CONF_COMMAND_TEMPLATE], self.hass
+        )
```

Comment:
Should we instantiate the MqttCommandTemplate in the schema validator instead, and just pass it a valid `hass` here?

Reply:
That would be a good improvement, and I think will also will simplify the code even more.